.gitignore

```diff
@@ -43,6 +43,8 @@ pip-log.txt
 nosetests.xml
 
 # generated examples
-/examples/boards/ppro/xilinx
+/examples/boards/papilio_pro/xilinx
```

Comment:
Yes, that's a great idea.
I've added
/examples/*/xilinx
as well as
/examples/*/*/xilinx
to gitignore.